### ISSUE TITLE
[Linux/Ubuntu 14.04] Cant pick a hero, nor can I move him if I eventually manage to pick him

### ISSUE THREAD

**Issue by grasdaggel** [ValveSoftware/Dota-2#73]
Played a round of Overthrown.

Not sure what caused this. Nvidia 331 driver, Linux 14.04

**Comment by kingalca** [comment]
http://imgur.com/I7n2Qiv

**Comment by ChipmunkV** [comment]
Possible duplicate: #48

**Comment by gdrewb-valve** [comment]
If you update to driver version 352 does that help?

**Comment by nopenopenope** [comment]
Hi, no that doesnt work for me. Now Reborn will not even start anymore. Updated to 352 right after I posted this here. 

Ah, if its confusing, this is my second github account. Im at work currently, just reviewing my issue.

**Comment by gdrewb-valve** [comment]
When you say it doesn't start do you mean it crashes or exits or does it just hang?  At what point during startup?  Do you get any console spew?  Can you post more complete system info?

**Comment by nopenopenope** [comment]
Following: updated to said driver, restarted Dota and selected Reborn. Dota client doesn't start, so I assume it throws an unexpected error and simply exits. However, after several restarts of my machine I eventually managed to see a stable window of Dota2 Reborn under my Linux Ubuntu 14.04 machine. 

So far, so good. Theres another bug I've encountered, but I'll open another ticket for that.

**Comment by Tele42** [comment]
@nopenopenope: After updating nvidia drivers, you do need to restart the system, or at a minimum, stop X, unload the previous nvidia kernel module, then modprobe the new kernel module, before starting X back up. When the kernel module does not match the userspace libraries, nvidia blocks all new opengl windows from starting, to prevent chaos.

Note: Reloading the nvidia kernel module is only relevent if the driver has been updated, and doing this seems poorly tested by nvidia with Fermi (GTX 4xx) and newer cards. It's easier to just tell the user to reboot.

**Comment by gdrewb-valve** [comment]
Should I go ahead and close this now?

**Comment by nopenopenope** [comment]
Yes, go ahead. Its all good now. Thanks man
